common/shared.c

```diff
@@ -2339,6 +2485,18 @@ void ft_parse_addr_opts(int op, char *optarg, struct ft_opts *opts)
 	case 'P':
 		opts->dst_port = optarg;
 		break;
+	case 'x':
```

Comment:
I'm hoping to use this for ubertest also which already uses 'x'. Could we change this to something else to distinguish it? Maybe 'b'? I don't think that's used anywhere?

Reply:
I've pulled this in locally.  I've updated the patch to use 'b' in place of 'x', plus simplified the getopt() handling to use an optional port argument.

Reply:
I have a patch I just posted which moves the out of band address exchange to a function and uses 'b' for the arg. Could you reapply your simplification of the getopt on my new patch? Thanks!